internal/controller/apiextensions/composite/transforms.go

```diff
@@ -354,6 +364,18 @@ func ResolveConvert(t v1.ConvertTransform, input any) (any, error) {
 	if to == v1.ConvertTransformTypeInt {
 		to = v1.ConvertTransformTypeInt64
 	}
+
+	var format v1.ConvertTransformFormat
+	if t.Format != nil {
+		format = *t.Format
+	}
+	switch format {
+	case "", v1.ConvertTransformFormatQuantity:
```

Comment:
nit: Should we define a default convertTransformFormat instead of an empty string for better readability? 
`v1.ConvertTransformFormatNone` or `v1.ConvertTransformFormatRaw` ?

Reply:
Good idea. I added a `none` value. That the controller will fall back to.